### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by victhevenot**
@jmburges @AnnJohn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+If we have a controller action:
+
+```ruby
+get '/' do 
+  erb: index
+```
+
+When the above controller action is triggered, and the `erb` method is called, it looks to see if there is a view titled `layout.erb`. If that file exists, it loads that content around the desired erb file, in this case `index.erb`
```
Comment (jmburges) [inline README.md]: can you add in what the resulting html will look like after the yield?

**Comment by jmburges** [comment]
small addition and :shipit:

File: README.md (hunk 1 above)
Comment (victhevenot) [inline README.md]: done

**Comment by victhevenot** [comment]
merged

**Pull request merged**